includes/install.php

```diff
@@ -391,6 +391,8 @@ function give_get_default_settings() {
 		// Stripe accounts.
 		'_give_stripe_get_all_accounts'               => [],
 
+		// Onboarding
+		'setup_page_enabled'                          => true,
```

Comment:
@kjohnson This value store in DB. I did not see in code that you have a setting in which `true` is one of option.

Reply:
@ravinderk I'm using `true`/`false` everywhere except for the advanced-settings (which maps to 1/0 to match the existing settings). See `src/Onboarding/Onboarding.php`